plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
```

Comment:
let's just be explicit with the rest of the `kwargs` like `filename`, `output_format`, etc